1.purpose_and_goals.md

```diff
@@ -22,13 +22,13 @@ Additional goals of the specification are to:
 
 Non-goals include:
 
-- Defining or prescribing specific development or operational workflows.
+- Defining or prescribing specific development or operational workflow.
 - Defining the schemas of operational resources, for example (but not limited
   to):
   - Secrets (secure, encrypted values)
   - Networks
   - Volumes
-- Describing or defining the runtimes themselves
+- Describing or defining the runtime themselves
```

Comment:
I think this was more grammatically correct with the "s" too, but "Describing or defining a runtime itself" would also work.